easy-traits/src/main.rs

```diff
@@ -0,0 +1,71 @@
+// Your task is to implement the trait
```

Comment:
This one is tricky because traits are useful to define behavior that is shared by multiple types that implement it. Defining a trait that is only implemented for one type defeat the whole purpose. Don't you think?